apps/analytics/urls.py

```diff
@@ -10,8 +10,10 @@
         views.get_participant_count, name='get_participant_count'),
     url(r'^challenge/(?P<challenge_pk>[0-9]+)/submission/(?P<duration>[A-Za-z]+)/count$',
         views.get_submission_count, name='get_submission_count'),
-    url(r'^challenge/(?P<challenge_pk>[0-9]+)/challenge_phase/(?P<challenge_phase_pk>[0-9]+)/count$',
+    url(r'^challenge/(?P<challenge_pk>[0-9]+)/challenge_phase/(?P<challenge_phase_pk>[0-9]+)/analytics$',
```

Comment:
Just a future note: Lets make sure that we add more challenge phase related analytics data to this endpoint only so that we just need to do one API call to get all the analytics related to a challenge phase.

Reply:
yes! I will make sure this.